crates/nargo/src/cli/mod.rs

```diff
@@ -119,7 +119,8 @@ pub fn start_cli() {
                 .arg(
                     Arg::with_name("witness_name")
                         .long("witness_name")
-                        .help("Write the execution witness to named file"),
+                        .help("Write the execution witness to named file")
+                        .takes_value(true),
```

Comment:
Can you elaborate on this change for the execute command only?

Reply:
Clap seems to be treating this option as a boolean flag rather than something which takes a string value. Adding this line allows us to pass a value which nargo will then pick up as the desired name of the witness file.

Alternatively we could make this a positional argument (I think by removing the `long` modifier?)